lib/Service/TableService.php

```diff
@@ -238,28 +259,29 @@ public function create(string $title, string $template, string $emoji, ?string $
 	/**
 	 * @noinspection PhpUndefinedMethodInspection
 	 *
-	 * @param int $id$userId
+	 * @param int $id $userId
 	 * @param string $title
-	 * @param string $emoji
+	 * @param string|null $emoji
 	 * @param string|null $userId
 	 * @return Table
 	 * @throws InternalError
 	 */
-	public function update(int $id, string $title, string $emoji, ?string $userId): Table {
-		$userId = $this->permissionsService->preCheckUserId($userId, false); // $userId is set
+	public function update(int $id, string $title, string $emoji = null, ?string $userId = null): Table {
```

Comment:
For consistency we could also have the title nullable so you can update only the emoji, but no blocker